apps/docs/src/app/platform/component-docs/platform-forms/multi-input/platform-mulit-input-example/platform-mulit-input-reactive-example.component.html

```diff
@@ -0,0 +1,29 @@
+<form #upload="ngForm" (ngSubmit)="onSubmit()">
+    <fdp-form-group [formGroup]="customForm">
+        <fdp-form-field
+            #ffmi1
+            label="Default Multi Input Field"
+            id="input-simple"
+            name="reactiveFormInput"
+            zone="zLeft"
+            rank="1"
+            placeholder="Field placeholder text"
+            [required]="true"
+        >
+            <fdp-multi-input
+                [formControl]="reactiveForm"
+                placeholder="Field placeholder text"
+                type="text"
+                id="fdp-id-reactive"
+                name="input1"
+                [dataSource]="_datasource"
+                displayKey="firstName"
+                [selectionMode]="'Multi'"
+                [status]="!hasError() ? 'error' : 'success'"
```

Comment:
setting of status should not be manual. According to validation rules, it will detect error, when validation fails.
we should be using some validation function here.

Reply:
Known Issue:
Form field is unable to recognise the Multi input component ,